.git-blame-ignore-revs

```diff
@@ -1,7 +1,3 @@
-# Automatically apply to git blames with `git config blame.ignorerevsfile .git-blame-ignore-revs`
-
-# Apply black formatting
-df271e3ebe0fd007b06177a3fda82f9139d111ce
-
-# Apply isort formatting
-7fe0c1d2475cb195d6fc5746da949ea9cd7bf3b4
```

Comment:
Why are we dropping these other commitishes from the ignore file. Seems we want to ignore them too right?

Reply:
oh wait i didnt even realize there was already a `.git-blame-ignore-revs` file oops... lemme fix. thanks for catching this :sweat_smile: